ui/src/components/statistics-card.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2024 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+import { Link, LinkOptions } from '@tanstack/react-router';
+
+import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card';
+
+type StatisticsCardProps = {
+  title: string;
+  icon?: React.ComponentType<{ className?: string }>;
+  value?: number | string;
+  resultsPath?: LinkOptions['to'];
+};
+
+export const StatisticsCard = ({
+  title,
+  icon: Icon,
+  value,
+  resultsPath,
+}: StatisticsCardProps) => {
+  return (
+    <Link to={resultsPath} className={resultsPath ? '' : 'cursor-default'}>
+      <Card className={resultsPath ? 'hover:bg-muted/50' : ''}>
```

Comment:
I'd do it with the `className` as props approach I suggested, as this renders a link even if `resultsPath` is not given.

Reply:
So to recap: move Link to parent page, then use className inside the component to cover for hovering action when it's a link?

Reply:
That's how I'd do it, but feel free to do it the other way if that makes more sense to you. If the component works as a generic link, it should expose `params` and `search` in addition to `to`.

Reply:
@mmurto redid it with link removed, would this now be ready to merge?